### DISCUSSION TITLE
Change NC_DB from a SQLite(Local) version [Feb 03]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#5073] [Discord Threads]
> 🙋‍♂️ **unkn**

**Comment by o1lab** [comment]
> 🙋‍♂️ **Dstala | NocoDB Committer**  
#### <@170964650865852416>    
#### If you wish to change location of SQLite file, you can do that & reconfigure NC_DB accordingly.   
####    
#### If you wish to change NC_DB from SQLite to another database (say MySQL), that wouldn't be possible without loss of existing meta.   
####    
#### If thats not what you are looking for, please add some more details. We can help.  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1059883010759073802)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
